### ISSUE TITLE
Search snippet include reST markup

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#192]
The results of a search include reST markup, instead of HTML. For example, this is the first result for a search in my documentation:

```
# Coding Guide
...uide ============ There are a number of unusual coding features, which developers working on |phc| should know about. Debugging --------- |phc| uses :program:`libtool`, which can make debugging tricky [#libtool_link]_. When you want to c...
```

Obviously, the kind of markup slipping through is :program:`libtool` or |phc| or [#libtool_link]_.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/192
- Originally reported by: Paul Biggar
- Originally created at: 2009-06-02T14:34:05.049

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2009-06-02 16:44:40+00:00_

Yes, this is expected -- the highlight is done in the source files.

It's of course not optimal, so I'm reclassifying it as an RFE.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2013-03-31 13:57:05+00:00_

Removing version: 0.6 (automated comment)

**Comment by deric** [comment]
+1 any progress on this?

**Comment by electronwill** [comment]
+1. It would be great if Sphinx search results displayed the text seen in the rendered page and not the source!

**Comment by tk0miya** [comment]
Nothing changed because we don't have enough time.
It would be nice if anyone sends us PR for this :-)

**Comment by tk0miya** [comment]
This was resolved in v2.0 (see #4022). Closing.
